pyproject.toml

```
@@ -24,7 +24,7 @@ include = [
 [tool.poetry.dependencies]
 python = "^3.7"
 click = ">=7.1, <9.0"
-pydantic = {version = ">= 1.8.0, != 1.9.*, >= 1.10.4", extras = ["dotenv"]}
+pydantic = {version = ">= 1.8.0, != 1.9.*, >= 1.10.4, ^1", extras = ["dotenv"]}
```

I didn't know you could mix the specifier styles like this actually - would the below be preferable?

```suggestion
pydantic = {version = ">= 1.8.0, != 1.9.*, >= 1.10.4, < 2", extras = ["dotenv"]}
```

---

@qduk I also prefer @glennmatthews suggestion. could you check?

---

That will work. Updated